tests/runtime/basic

```diff
@@ -158,6 +158,22 @@ EXPECT hardware:cache-misses:
 EXPECT tracepoint:xdp:mem_connect
 TIMEOUT 5
 
+NAME it lists given search pattern
+RUN {{BPFTRACE}} -l hardware:cache-misses
+EXPECT hardware:cache-misses:
+TIMEOUT 5
+
+NAME it lists given search pattern
+RUN {{BPFTRACE}} -l runtime/scripts/hello_world.bt
+EXPECT special::
```

Comment:
Super nit: can we use a script that doesn't only have a single special probe that doesn't print quite right ? lol